rancher/rancher
# [2.7.0] Remove v1 istio

## Issue: https://github.com/rancher/rancher/issues/37327 

 
## Problem

Istio v1 chart was deprecated in 2.5.0, and therefore will be EOL as of 2.7.0. It should be removed/disabled
## Solution

Remove istio v1 components from rancher and supporting repos.
## Testing


## Engineering Testing
### Manual Testing

Provisioned Rancher server and enabled the legacy feature flag. Select Multicluster apps -> Launch apps, istio should not be visible.
### Automated Testing


## QA Testing Considerations

 Upgrade scenario for the rancher server to the new rancher server which has v1 istio removed.
### Regressions Considerations
